src/swap/handle_swap_sign_transaction.h

```diff
@@ -0,0 +1,28 @@
+/*******************************************************************************
+ *   Tron Ledger Wallet
+ *   (c) 2023 Ledger
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ ********************************************************************************/
+#pragma once
+
+#include <stdbool.h>
+
+#define MAX_SWAP_TOKEN_LENGTH 15
```

Comment:
where does this value come from ? Does it include terminal 0 ?
We could add this in the shared api file